go/vt/tabletserver/tabletenv/tabletenv.go

```diff
@@ -29,10 +29,27 @@ var (
 	WaitStats = stats.NewTimings("Waits")
 	// KillStats shows number of connections being killed.
 	KillStats = stats.NewCounters("Kills", "Transactions", "Queries")
-	// InfoErrors shows number of various non critical errors happened.
-	InfoErrors = stats.NewCounters("InfoErrors", "Retry", "DupKey")
 	// ErrorStats shows number of critial erros happened.
-	ErrorStats = stats.NewCounters("Errors", "Fail", "TxPoolFull", "NotInTx", "Deadlock", "Fatal")
+	ErrorStats = stats.NewCounters(
+		"Errors",
```

Comment:
NewCounters is actually a misleading constructor. `Errors` is the main tag, and the rest are sub-categories.